packages/webpack-cli/lib/utils/index.js

```diff
@@ -1,6 +1,22 @@
+let colorette = null;
```

Comment:
Why we need this?

Reply:
To avoid recreating colorette instance on every getter call. We could simply import default colouring operations, but these would default to always on and ignore configuration overrides.

Reply:
We decided to classify the always-color behavior as a bug after all (https://github.com/jorgebucaran/colorette/pull/72). If you don't really care about the new `createColors` API, you may continue using Colorette as you were before, just upgrade to `2.0.5` or later.